csharp/src/Ice/InputStream.cs

```diff
@@ -1200,11 +1200,13 @@ internal string ReadFacet()
         /// </summary>
         /// <param name="replyStatus">The reply status.</param>
         /// <returns>The exception read from the stream.</returns>
-        internal Exception ReadSystemException11(ReplyStatus replyStatus)
+        internal DispatchException ReadSystemException11(ReplyStatus replyStatus)
```

Comment:
Perhaps we should consider renaming this to just `Read11Exception`... RFE or unknown exceptions weren't really system exceptions in 3.7 and there's no system exceptions in 4.0?